docker-compose.yml

```
@@ -290,3 +290,25 @@ services:
     environment:
       - NODE_ENV=development
       - COUNTRY_CONFIG_URL=http://countryconfig:3040
+  
+  data-seeder:
+    image: ghcr.io/opencrvs/ocrvs-data-seeder:${VERSION}
+    #platform: linux/amd64
```

```suggestion
    #platform: linux/amd64

    deploy:
      replicas: 0
```

data seed should be executed only once, we need it here only for build like base image.

---

got it
i was on confusion about this